charts/sysdig/Chart.yaml

```diff
@@ -15,4 +15,4 @@ name: sysdig
 sources:
 - https://app.sysdigcloud.com/#/settings/user
 - https://github.com/draios/sysdig
-version: 1.16.25
+version: 1.16.26
```

Comment:
I think we shall not update anymore the `sysdig` chart since it has been deprecated

Reply:
Should we ditch it then?